File: responses/K3H7Q3-DPP/2_reverse_digits.py
# you can write to stdout for debugging purposes, e.g.
# print("this is a debug message")

def solution(X):
    # write your code in Python
    ans = str(X)
    if ans[0] == "-":
        return int("-" + ans[:0:-1])
    else:
        return int(ans[::-1])
    pass

File: responses/VWDVV4-W6X/2_reverse_digits.js
// you can write to stdout for debugging purposes, e.g.
// console.log('this is a debug message');

function solution(X) {
    // write your code in JavaScript
    var strVal = X.toString();
    var mod = 1;
    strVal = strVal.split("").reverse().join("");
    if (X < 0) {
        mod = -1;
    }

    return parseInt(strVal) * mod;
}